server/models/Interview.mjs

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+
+const interviewSchema = new mongoose.Schema({
+  interview_id: { type: Number, required: true, unique: true },
```

Comment:
MongoDB creates an object ID automatically, so we don't need our own interview_id.

Reply:
Good tip, removing the object IDs for each schema (except in the cases where it's referencing a different object).